samples/build.gradle

```diff
@@ -19,7 +19,8 @@ android {
 }
 
 dependencies {
-  compile 'com.dlazaro66.qrcodereaderview:qrcodereaderview:2.0.0'
+  //compile 'com.dlazaro66.qrcodereaderview:qrcodereaderview:2.0.0'
+  compile project (":qrcodereaderview")
```

Comment:
I'm finally setting this to link the local project instead of the remote maven repository